### DISCUSSION TITLE
Thanks for all your work and a quick question

### DISCUSSION THREAD

**Discussion by bpeatman** [prymitive/karma#4335] [General]
First I want to say thanks for everyones work on this. We live and die by this tool and it makes seeing what is going on across all our environments much better.

I have a question about the gui I cant seem to find anywhere. Some of the cards for alerts have  colors in the dashed lines that are at the top of every card. Are these alerts that are coming in and then clearing right away? 

![image](https://user-images.githubusercontent.com/12719969/170728518-18e87d17-3c8e-4b82-8fa5-2562c623bba4.png)

**Comment by prymitive** [comment]
https://github.com/prymitive/karma#alert-history
